docs/source/qvm.rst

```diff
@@ -353,7 +359,10 @@ Providing Your Own Quantum Processor Topology
 It is simple to provide your own quantum processor topology as long as you can give your qubits each a number,
 and specify which edges exist. Here is an example, using the topology of our 16Q chip (two octagons connected by a square):
 
-.. code:: python
+..
+    Will this need to be updated for Ankaa?
```

Comment:
That is a good question, this example is clearly inspired by the Aspen topology. I'm not sure if it _needs_ to change though, since this section is QPU agnostic and just serves to show a way to provide a custom topology, which it still does.

I would say create an issue, and we can prioritize it accordingly.

Reply:
https://github.com/rigetti/pyquil/issues/1580